gradle/publish-bintray.gradle

```diff
@@ -89,8 +89,8 @@ publishing {
                 break
         }
 
-        // disable metadata everywhere, but in native modules
-        if (type == 'maven' || type == 'metadata' || type == 'jvm' || type == 'js') {
+        // disable metadata everywhere, but in native and js modules
+        if (type == 'maven' || type == 'metadata' || type == 'jvm') {
```

Comment:
Isn't it going to break our releases with Kotlin 1.3.70? Do we really have to publish metadata if IR BE is not here?

Reply:
> Do we really have to publish metadata if IR BE is not here?
We don't

> Isn't it going to break our releases with Kotlin 1.3.70?
I assumed adding metadata wouldn't break stuff. @ilgonmic please take a look

Reply:
Since Gradle 5.3 it can consume metadata for dependency from repo. So for this version range it should not break release.
But this check can be covered with plus one check on 1.4 release, and not publish metadata for 1.3.70 releases